.github/workflows/client-ethereum.yml

```diff
@@ -41,40 +41,40 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           target: gobuild
-          tags: go-build-env
+          tags: go-build-env-ethereum
           build-args: |
           # REVISION=${{ github.sha }} TODO: set revision only when we want to publish the image
           # VERSION= ? TODO: Configure version, sample: 1.7.6
           load: true # load image to local registry to use it in next steps
-          cache-from: type=local,src=/tmp/.buildx-cache
-          cache-to: type=local,dest=/tmp/.buildx-cache
+          cache-from: type=local,src=/tmp/.buildx-ethereum-cache
+          cache-to: type=local,dest=/tmp/.buildx-ethereum-cache
 
       - name: Create test results directory
-        run: mkdir test-results
+        run: mkdir test-results-ethereum
           
       - name: Run Go tests
         run: |
           docker run \
-            --volume $GITHUB_WORKSPACE/test-results:/mnt/test-results \
+            --volume $GITHUB_WORKSPACE/test-results-ethereum:/mnt/test-results-ethereum \
             --workdir /go/src/github.com/keep-network/keep-ecdsa \
-            go-build-env \
-            gotestsum --junitfile /mnt/test-results/unit-tests.xml
+            go-build-env-ethereum \
+            gotestsum --junitfile /mnt/test-results-ethereum/unit-tests-ethereum.xml
 
       - name: Publish unit test results
         uses: EnricoMi/publish-unit-test-result-action@v1
         if: always()
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          files:  ./test-results/unit-tests.xml
-          check_name: Go Test Results # name under which test results will be presented in GitHub (optional)
+          files:  ./test-results-ethereum/unit-tests-ethereum.xml
+          check_name: Ethereum Client Test Results # name under which test results will be presented in GitHub (optional)
           comment_on_pr: false # turns off commenting on Pull Requests
```

Comment:
I don't think there is a need to add `-ethereum` suffix to `test-results` folder and `unit-tests.xml` file. We don't populate the files anywhere outside the workflow, they 'live' within the runner, only for the duration of the job execution.

Reply:
See https://github.com/keep-network/keep-ecdsa/pull/712#discussion_r587400670

Reply:
Please see https://github.com/keep-network/keep-ecdsa/commit/6266c7a884d8ab3cdd1d68c65c03b4a7df94356f